app/vmalert/group_test.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"context"
 	"fmt"
+	"github.com/stretchr/testify/assert"
```

Comment:
We don't use extra packages for testing in VictoriaMetrics project. Adding it only here seems like an overkill.